### DISCUSSION TITLE
0.76.3-0 release is now available

### DISCUSSION THREAD

**Discussion by douglowder** [react-native-tvos/react-native-tvos#835] [Announcements]
## What's Changed
* Changes from RN core [0.76.3 release](https://github.com/facebook/react-native/releases/tag/v0.76.3)
* Bump cross-spawn from 7.0.3 to 7.0.6 by @dependabot in https://github.com/react-native-tvos/react-native-tvos/pull/828
* fix: TextInput regression with new native events and old arch by @douglowder in https://github.com/react-native-tvos/react-native-tvos/pull/834

**Full Changelog**: https://github.com/react-native-tvos/react-native-tvos/compare/v0.76.2-0...v0.76.3-0

<hr /><em>This discussion was created from the release <a href='https://github.com/react-native-tvos/react-native-tvos/releases/tag/v0.76.3-0'>0.76.3-0</a>.</em>

**Comment by MobileMon** [comment]
button presses no longer working on apple tv

basic pressing the middle/enter button not working with touchable opacity

**Reply by MobileMon** [reply]
@douglowder - are you seeing this issue?

**Reply by saadlatif75** [reply]
@MobileMon Have you work with epg in react native tvos

**Reply by douglowder** [reply]
I am not seeing this issue.
